ui/app/adapters/sync/association.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import ApplicationAdapter from 'vault/adapters/application';
+
+export default class SyncDestinationAdapter extends ApplicationAdapter {
+  namespace = 'v1/sys/sync/destinations';
```

Comment:
Looks like these need to be updated to reference associations

Reply:
Hmm am I missing something? The docs have this as the endpoint for reading associations `/sys/sync/destinations/:type/:name/associations`. The other endpoints just add `set` and `remove` on the end. Unless I'm maybe looking at out of date docs.

Reply:
Following up from our conversation for posterity. Since new endpoints have been added that break from the namespace defined here, we may have to iterate on this but until we determine if we will be using those endpoints it can stay as is.